ui/components/component-library/text-field/README.mdx

```diff
@@ -1,4 +1,4 @@
-import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+import { Story, Canvas, ArgTypes } from '@storybook/addon-docs';
```

Comment:
`ArgsTable` has been deprecated in favor of `ArgTypes` there are a few more but we can address them in a storybook clean up